docs/en/sql-reference/sql-statements/Administration/ADMIN_SET_PARTITION_VERSION.md

```diff
@@ -6,7 +6,9 @@ displayed_sidebar: "English"
 
 ## Description
 
-Manually setting the partition version is a dangerous operation and is only used when there is a problem with the cluster metadata. Under normal circumstances, the partition version will automatically be consistent with the tablet version.
+Sets a partition to a specific data version.
+
+Note that manually setting the partition version is a critical operation and is recommended only when problems occur in the cluster metadata. In normal circumstances, the version of a partition is automatically set consistent with that of the tablet within.
```

Comment:
Note that manually setting the partition version is a high-risk operation

Reply:
In normal circumstances, the version of a partition is consistent with that of the tablets within.